addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -487,7 +487,7 @@
 
             min-height: initial;
             height: 100%;
-            overflow: scroll hidden !important;
+            overflow: visible !important;
```

Comment:
I am wondering if the issue is still there. I cannot reproduce it. Can you check to be sure?

Reply:
Hello @xavierbol 
I have checked, but the issue persists. this is the screenshot as a reference
![image](https://github.com/odoo/odoo/assets/136676190/9d3c2cba-da39-46be-8e45-e1614015bd0c)

Reply:
Ah yeah indeed, my bad

Reply:
@rfr-odoo maybe?

Reply:
```patch
-            overflow: scroll hidden !important;
+            overflow: visible !important;
```
Breaks the snapping feature of the kanban on the horizontal axis (`scroll-snap-type: x` CSS rule).